### PR TITLE
Correction: Use HTTPS instead of HTTP for links

### PR DESCRIPTION
**Pull request by mechanomi**
## Summary of the changes / Why this is an improvement

see crate/tech-writing-domain#316

### CODE REVIEW

File: docs/clustering/multi-node-setup.rst [hunk 1]
```diff
@@ -461,8 +461,8 @@ Edit the `transport.tcp.port`_ setting in your `configuration`_ file:
     `More information about port settings`_
```
Comment (mkleen) [inline docs/clustering/multi-node-setup.rst:461]: This section seems broken to me.

Comment (mechanomi) [reply docs/clustering/multi-node-setup.rst]: @mkleen oh you're totally right. pushed a fixup. thx!

Comment (mkleen) [reply docs/clustering/multi-node-setup.rst]: Screenshot 2020-12-18 at 14 47 59

Comment (mkleen) [reply docs/clustering/multi-node-setup.rst]: This is what i meant with broken section.

Comment (mechanomi) [reply docs/clustering/multi-node-setup.rst]: @mkleen I think this must be an issue with your local dev setup. can you try running `make reset` inside `docs/` and then try again?

you can view the correct output on RTD here:

https://crate-howtos--263.org.readthedocs.build/en/263/clustering/multi-node-setup.html

this (the RTD build) looks fine to me

Comment (mkleen) [reply docs/clustering/multi-node-setup.rst]: oh you are right, sorry for that.

**Review by mkleen — APPROVED** [review]

**Pull request merged**